students/sungho-moon/user/models.py

```diff
@@ -1,3 +1,12 @@
-from django.db import models
+from django.db              import models
 
 # Create your models here.
+
+class USER(models.Model):
```

Comment:
class 이름은 카멜케이스가 적용됩니다.
```suggestion
class User(models.Model):
```

Reply:
수정완료했습니다